swift-sdk/Internal/in-app/InAppMessageParser.swift

```diff
@@ -81,41 +81,78 @@ struct InAppMessageParser {
             return .failure(.parseFailed(reason: "no messageId", messageId: nil))
         }
         
-        guard let contentDict = json[JsonKey.InApp.content] as? [AnyHashable: Any] else {
-            return .failure(.parseFailed(reason: "no content in json payload", messageId: messageId))
-        }
-        
-        let content: IterableInAppContent
+        let jsonOnly = (json[JsonKey.InApp.jsonOnly] as? Int ?? 0) == 1
```

Comment:
I would add that comment just to clarify what's happening. the one liner is hard to read in one pass.
```suggestion
        // Check if the jsonOnly key is present and is set to 1 (true)
        let jsonOnly = (json[JsonKey.InApp.jsonOnly] as? Int ?? 0) == 1
```